src/docs/get-started/install/_ios-setup.md

```diff
@@ -82,9 +82,10 @@ to set up physical device deployment in Xcode.
  1. Install and set up CocoaPods by running the following commands:
 
     ```terminal
-    $ sudo gem install cocoapods
+    $ sudo gem install cocoapods -v 1.7.5
     $ pod setup
     ```
+    {{site.alert.warning}} Versions of CocoaPods greater than 1.7.5 are not compatible with the most recent stable version of Flutter. See [this issue](https://github.com/flutter/flutter/issues/41253) for details and a forthcoming fix. {{site.alert.end}}
```

Comment:
Can you break up the lines so that each line no more than 80 chars? Also, indent the lines inside the warning by two spaces per line. Also, the link text must be more descriptive, which is an accessibility requirement. We use macros for GitHub links. Finally, I think a note (green) is sufficient for this text, rather than a warning (yellow).

How about this:

{{site.alert.note}}
  The `pod setup` command fails for versions of CocoaPods greater
  than 1.7.5.   For more information, see [issue 41252][].
{{site.alert.end}}

[issue 41252]: {{site.github}}/flutter/flutter/issues/41253